ko/lessons/basics/enum.md

```diff
@@ -180,11 +182,11 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
 [%{count: 1}, %{count: 4}]
 ```
 
-### uniq
+### uniq_by
```

Comment:
This one is the only update this time.
Others just trimmed the sentences.

Reply:
But `uniq_by/2` needs 2nd argument as anonymous function.
Also the following explanation and code block should be corrected like in English:

`uniq/1` -> `uniq_by/2`
`iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])` -> `iex> Enum.uniq_by([1, 2, 3, 2, 1, 1, 1, 1, 1], fn x -> x end)`

Reply:
I mean 185, 187 and 190 lines.
Updated aleady. 💪

Reply:
Oh.. you've already done 🙀 
Sorry for my carelessness 😂  LGTM!